luatest/server.lua

```diff
@@ -158,7 +284,8 @@ function Server:restart(params)
         alias = '?string',
 
         coverage_report = '?boolean',
-    })
+    }, {wait_until_ready = '?boolean'})
```

Comment:
Why isn't this `wait_until_ready` a part of `params`? Also, the trailing `)` should probably be on a separate line.

Reply:
Because these are different entities. 

`params` - New parameters to restart the server with.
`opts` - Options for the `start()` function that is inside the `restart()` function.

Reply:
Oh, I get it.